Sources/Purchasing/StoreKitAbstractions/SK2StoreProduct.swift

```diff
@@ -58,7 +65,12 @@ internal struct SK2StoreProduct: StoreProductType {
         return offers?.first?["currencyCode"] as? String
     }
 
-    var price: Decimal { underlyingSK2Product.price }
+    var price: Decimal {
+        // See: https://github.com/RevenueCat/purchases-ios/issues/2207
+        // it seems like StoreKit 2 doesn't actually round decimals in a price-friendly fashion, so we do it
```

Comment:
See the words from the [StoreKit whisperer](https://stackoverflow.com/a/47445051/5074358)